8Knot/pages/repo_overview/repo_overview.py

```diff
@@ -53,17 +49,14 @@
                     width=3,
                 ),
             ],
-            justify="center",
-            align="center",
-            style={"marginBottom": "1%"},
+            className="visualization-row",
```

Comment:
Using the visualization row styling on the drop down caused a format change/issue. The drop down should be next to the text with both in the middle
<img width="1405" height="118" alt="Screenshot 2025-10-03 at 1 07 56 PM" src="https://github.com/user-attachments/assets/81cbceb6-1fc2-4241-9229-44f661ac14d1" />